doc/rst/lib_otpinfo.rst

```diff
@@ -0,0 +1,52 @@
+
+################################
+lib_otpinfo: OTP reading library
+################################
+
+************
+Introduction
+************
+
+``lib_otpinfo`` provides functions to read data from the OTP memory of an xCORE device.
```

Comment:
I think first usage of OTP should be fully expanded (one time programmable)

Reply:
done